geode-core/src/integrationTest/java/org/apache/geode/internal/jta/functional/CacheJUnitTest.java

```diff
@@ -341,7 +344,7 @@ public void testScenario4() throws Exception {
       String current_fullpath = jtaObj.currRegion.getFullPath();
       assertEquals(
           "failed retrieving current region fullpath after doing getRegionFromCache(region1)",
-          "/" + DEFAULT_RGN + "/region1", current_fullpath);
+          SEPARATOR + DEFAULT_RGN + "/region1", current_fullpath);
```

Comment:
Did you miss this one?, shouldn't it be `SEPARATOR + DEFAULT_RGN + SEPARATOR + "region1"`?